db/models.py

```diff
@@ -1 +1,52 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACES = [
+        ("elf", "Elf race"),
+        ("dwarf", "Dwarf race"),
+        ("human", "Human race"),
+        ("ork", "Ork race")
+    ]
+
+    name = models.CharField(
+        max_length=255,
+        unique=True,
+        choices=RACES
+    )
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.CASCADE,
+        related_name="skill_set"
+    )
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.CASCADE,
+        related_name="player_race"
```

Comment:
It must be `players` because when we have a race instance we will access players like so:
`race.players`